.release-please-manifest.json

```diff
@@ -1,4 +1,4 @@
 {
   "packages/core": "2.16.0",
-  "packages/tokens": "1.2.0"
+  "packages/tokens": "2.0.0"
```

Comment:
question: why do we need to commit this?

Reply:
@Natalie9 we can't commit it or we will be problems about ci/cd